data/nginx/vhost.d/nextcloud.local_location_override

```diff
@@ -0,0 +1,8 @@
+location / {
```

Comment:
Doesn't this need to go in a file jsut called nextcloud.local https://github.com/nginx-proxy/nginx-proxy/tree/main/docs#per-virtual_host or is it possible to have multiple?

Reply:
yeah that didn't seem to work
we used this: https://github.com/nginx-proxy/nginx-proxy/tree/main/docs#overriding-location-blocks but the `location /` was also removed from the main config
maybe `_location` would allow to add just additional locations, will test